fedimint-api/src/db/mod.rs

```diff
@@ -81,8 +108,8 @@ pub trait IDatabaseTransaction<'a>: 'a {
 
     fn rollback_tx_to_savepoint(&mut self);
 
-    // Ideally, avoid using this in fedimint client code as not all database transaction
-    // implementations will support setting a savepoint during a transaction.
+    /// Ideally, avoid using this in fedimint client code as not all database transaction
```

Comment:
This is not written in a rustdoc style, while it just became one. :)

Reply:
I think this was suggested by @elsirion in a different PR. Should I just keep it as two slashes instead so that its not included in the rustdoc?

Reply:
It's good to do it here, it just it should have a proper `Start a snapshot that be restored with ...` as a first line. Then maybe `*Warning:* ...` in a next line of test or something like that.

Reply:
Gotcha, comment should be better now.